samples/boards/nrf52/power_mgr/src/power.c

```diff
@@ -1,95 +0,0 @@
-/*
- * Copyright (c) 2018 Nordic Semiconductor ASA.
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-#include "sample.h"
-
-/*
- * Application defined function for doing any target specific operations
- * for low power entry.
- */
-void sys_pm_notify_power_state_entry(enum power_states state)
```

Comment:
There is no other example how to use entry/exit hooks, but they can be useful for example to set dcdc converter mode.

Reply:
That's a fair point, but it should probably be a separate example with explanatory context.  I'm not clear on why I'd want to do that through power state events.